### PR TITLE
store: Fix Zookeeper single Key watch

### PR DESCRIPTION
**Pull request by abronan**
Fix Zookeeper single key `Watch` the same way than `WatchTree`.

ping @aluzzardi @vieux 

Signed-off-by: Alexandre Beslic <abronan@docker.com>

### CODE REVIEW

**Comment by vieux** [comment]
LGTM

File: pkg/store/zookeeper.go [hunk 1]
```diff
@@ -131,9 +131,13 @@ func (s *Zookeeper) Watch(key string, stopCh <-chan struct{}) (<-chan *KVPair, e
 	go func() {
 		defer close(watchCh)
 
-		// GetW returns the current value before setting the watch.
-		watchCh <- &KVPair{key, resp, uint64(meta.Version)}
+		// Get returns the current value before setting the watch.
```
Comment (aluzzardi) [inline pkg/store/zookeeper.go]: This is not true anymore, right?

File: pkg/store/zookeeper.go [hunk 2]
```diff
@@ -131,9 +131,13 @@ func (s *Zookeeper) Watch(key string, stopCh <-chan struct{}) (<-chan *KVPair, e
...
+		watchCh <- pair
 		for {
+			_, _, eventCh, err := s.client.GetW(fkey)
```
Comment (aluzzardi) [inline pkg/store/zookeeper.go]: How does `GetW` work?

Comment (aluzzardi) [inline pkg/store/zookeeper.go]: Meaning: it doesn't return the current value, only a watch right?

Comment (abronan) [inline pkg/store/zookeeper.go]: It's a one time watch thus the necessity to initialize it at each turn (If you want to keep watching events on that key).

File: pkg/store/zookeeper.go (hunk 1 above)
Comment (abronan) [inline pkg/store/zookeeper.go]: It is, we still send back the first value to the channel but this time using a simple `Get` ;)

Comment (aluzzardi) [inline pkg/store/zookeeper.go]: Yep, but it doesn't:

> before setting the watch

Comment (abronan) [inline pkg/store/zookeeper.go]: Oh yeah I see ;) I will change the comment.

**Comment by aluzzardi** [comment]
LGTM

**Pull request merged**